examples/deployments/systemd-service/chroma-cli.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = Chroma Service
+After = network.target docker.service
+Requires = docker.service
+
+[Service]
+Type = forking
+User=root
+Group=root
+WorkingDirectory = /home/admin/chroma
+ExecStart = /usr/bin/docker compose up -d
+ExecStop = /usr/bin/docker compose down
```

Comment:
It looks like these are still the docker commands? Are they meant to be?

Reply:
Nope, let me update.